mibitof/tests/test_tiff.py

```diff
@@ -0,0 +1,206 @@
+"""Tests for mibitof.tiff
```

Comment:
Did you make any changes in these files or did you just copy them over? (would be easier to review if it is the latter)

Reply:
They are just copy and pasted over.